### ISSUE TITLE
DensePose: Dense Human Pose Estimation In The Wild

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#543]
Güler, Rıza Alp, Neverova, Natalia, Kokkinos, Iasonas

In this work, we establish dense correspondences between RGB image and a
surface-based representation of the human body, a task we refer to as dense
human pose estimation. We first gather dense correspondences for 50K persons
appearing in the COCO dataset by introducing an efficient annotation pipeline.
We then use our dataset to train CNN-based systems that deliver dense
correspondence 'in the wild', namely in the presence of background, occlusions
and scale variations. We improve our training set's effectiveness by training
an 'inpainting' network that can fill in missing groundtruth values and report
clear improvements with respect to the best results that would be achievable in
the past. We experiment with fully-convolutional networks and region-based
models and observe a superiority of the latter; we further improve accuracy
through cascading, obtaining a system that delivers highly0accurate results in
real time. Supplementary materials and videos are provided on the project page
this http URL

https://arxiv.org/abs/1802.00434
